macros/AddonSidebar.ejs

```diff
@@ -12,145 +12,320 @@ function currentPageIsUnder(root) {
+    'WebExtensions/Porting_a_legacy_Firefox_add-on': 'Porting a legacy Firefox extension',
+    'WebExtensions/Embedded_WebExtensions': 'Embedded WebExtensions',
+    'WebExtensions/Comparison_with_the_Add-on_SDK': 'Comparison with the Add-on SDK',
+    'WebExtensions/Comparison_with_XUL_XPCOM_extensions': 'Comparison with XUL/XPCOM extensions',
+    'WebExtensions/Chrome_incompatibilities': 'Chrome incompatibilities',
+    'WebExtensions/Differences_between_desktop_and_Android': 'Differences between desktop and Android',
+    'WebExtensions#Firefox_workflow': 'Firefox workflow',
+    'WebExtensions/User_experience_best_practices': 'User Experience',
+    'WebExtensions/Temporary_Installation_in_Firefox': 'Temporary Installation in Firefox',
+    'WebExtensions/Debugging': 'Debugging',
+    'WebExtensions/Testing_persistent_and_restart_features': 'Testing persistent and restart features',
+    'WebExtensions/Developing_WebExtensions_for_Firefox_for_Android': 'Developing for Firefox for Android',
+    'WebExtensions/Getting_started_with_web-ext': 'Getting started with web-ext',
+    'WebExtensions/web-ext_command_reference': 'web-ext command reference',
+    'WebExtensions/WebExtensions_and_the_Add-on_ID': 'Extensions and the Add-on ID',
+    'WebExtensions/Request_the_right_permissions': 'Request the right permissions',
+    'WebExtensions/API': 'JavaScript APIs',
+    'WebExtensions/manifest.json': 'Manifest keys',
+    'Themes': 'Themes',
+    'Themes/Theme_concepts': 'Browser themes',
+    'Themes/Theme_concepts_': 'Browser theme concepts',
+    'Themes/Lightweight_themes': 'Lightweight themes',
+    'Themes/Lightweight_Themes/FAQ': 'Lightweight themes FAQ',
+    'Distribution': 'Publishing and Distribution',
+    'Distribution/Publishing': 'Publishing add-ons',
+    'Distribution/Signing': 'Signing and distribution overview',
+    'WebExtensions/Package_your_extension_': 'Package your extension',
+    'Distribution/Submitting_an_add-on': 'Submit an add-on',
+    'Source_Code_Submission': 'Source code submission',
+    'Distribution/Resources_for_publishers': 'Resources for publishers',
+    'Listing': 'Creating an appealing listing',
+    'AMO/Policy/Reviews': 'Review policies',
+    'AMO/Policy/Agreement': 'Developer agreement',
+    'AMO/Policy/Featured': 'Featured add-ons',
+    'Distribution/Retiring_your_extension': 'Retiring your extension',
+    'WebExtensions/Alternative_distribution_options': 'Distributing add-ons',
+    'WebExtensions/Alternative_distribution_options/Sideloading_add-ons': 'For sideloading',
+    'WebExtensions/Alternative_distribution_options/Add-ons_for_desktop_apps': 'For desktop apps',
+    'WebExtensions/Alternative_distribution_options/Add-ons_in_the_enterprise': 'For an enterprise',
+    'Community_and_Support': 'Community and Support',
+    'Channels': 'Channels',
+    'Blog': 'Add-ons blog',
+    'Forums': 'Add-on forums',
+    '#Contact_us': 'Contact us'
+  },
+  'fr-FR': {
```

Comment:
I know almost nothing about the way Kuma handles locales, but I think this needs to be just "fr".

If I try your macro, on, say, "http://llocalhost:8000/fr-FR/docs/Web/CSS", then I get redirected to "http://localhost:8000/fr/docs/Web/CSS" and see the en-US strings (I guess this is used as the fallback.

If I change line 102 to just "fr" then I see the French strings.

Reply:
Fixed :+1: